openshift/keycloak-db.app.yaml

```diff
@@ -31,7 +31,13 @@ objects:
       type: Recreate
       recreateParams:
         timeoutSeconds: 600
-      resources: {}
+      resources:
+        requests:
+          cpu: 50m
+          memory: 100Mi
+        limits:
+          cpu: 500m
```

Comment:
it could even reach spikes of 2k millicores :/. If we keep one single pod we might be careful with the limits